.github/workflows/test.yml

```diff
@@ -23,13 +23,20 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-      - name: Use Node.js ${{ matrix.node-version }}
+      - name: Installing with latest Node.js
```

Comment:
This is so we can use workspaces, which require node 16, but still run tests in node 12